src/js/store.js

```diff
@@ -9,7 +9,7 @@ import routerMiddleware from "react-router-redux/middleware";
 import appHistory from "tools/appHistory";
 import { composeWithDevTools } from "redux-devtools-extension";
 
-
+//__DEV_ENV_HERE__ = "development"

```

Comment:
Remove this comment. Gulpfile in `develop` should be updated. Just make sure you have the correct `.env.`